vet-service/src/test/java/com/petclinic/vet/presentationlayer/VetControllerIntegrationTest.java

```diff
@@ -1375,6 +1375,37 @@ void deleteAnEducationForVet_WithValidId_ShouldSucceed() {
                 .expectBody();
     }
 
+    @Test
+    void addEducationToAVet_WithValidValues_shouldSucceed(){
+        Publisher<Education> setup = educationRepository.deleteAll()
+                .thenMany(educationRepository.save(education1));
+
+        StepVerifier
+                .create(setup)
+                .expectNextCount(1)
+                .verifyComplete();
+
+        client.post()
+                .uri("/vets/" + vet.getVetId() + "/educations")
+                .accept(MediaType.APPLICATION_JSON)
+                .contentType(MediaType.APPLICATION_JSON)
+                .bodyValue(education2)
+                .exchange()
+                .expectStatus().isOk()
```

Comment:
After ResponseEntity has been implemented in a later user story, this line could be replaced with ".expectStatus().isCreated()"